src/components/Technologies/Technologies.tsx

```diff
@@ -134,10 +137,17 @@ export default function Technologies({ data, repositories }: TechnologiesProps)
   }
 
   const setSelectedSearchFilters = (selectedSearchFilters: Record<string, any>) => {
-    setSelectedFilters((prevState) => ({
-      ...prevState,
+    const updatedFilters = {
+      ...selectedFilters,
       ...selectedSearchFilters
-    }));
+    };
+    localStorage.setItem("selectedFilters", JSON.stringify(updatedFilters));
```

Comment:
Better make this a bit more specific "packList.selectedFilters" and make it a constant (to be used in both places when you set or get the value in the local storage)
and include the search value in these filters. There is no need for two different storage items

Reply:
search is still a filter